src/plugins/mer/meremulatormodedialog.cpp

```diff
@@ -204,7 +190,8 @@ void MerEmulatorModeDialog::execDialog()
         goto end;
     }
 
-    m_emulator.data()->setDeviceModel(m_ui->deviceModelComboBox->currentData().toString());
+    MerSettings::setCurrentDeviceModel(m_emulator.data()->id(), m_ui->deviceModelComboBox->currentData().toString());
+    m_emulator.data()->setDeviceModel(MerSettings::currentDeviceModel(m_emulator.data()->id()));
```

Comment:
It is related to that you said me there will be more than 1 emulator soon. So I added QMap that bond together emulator id and selected device model for it.

Reply:
I mean we should save MerEmulatorDevice::deviceModel in MerEmulatorDevice::toMap and load in MerEmulatorDevice::fromMap instead of save/load it as a MerSettings property.

Reply:
After closer look, it is already done, so MerSettings::setCurrentDeviceModel removed.